### PR TITLE
Switch from Debug -> Verbose

### PR DESCRIPTION
**Pull request by GregTheGreek**

### CODE REVIEW

File: packages/lodestar/src/logger/winston.ts [hunk 1]
```diff
@@ -18,7 +18,7 @@ export class WinstonLogger implements ILogger {
       ...options,
     };
     this.winston = createLogger({
-      level: LogLevel[LogLevel.debug], // log level switching handled in `createLogEntry`
+      level: LogLevel[LogLevel.verbose], // log level switching handled in `createLogEntry`
```
Comment (mpetrunic) [inline packages/lodestar/src/logger/winston.ts:21]: should be just `LogLevel.verbose` or `options.level`

**Review by mpetrunic — CHANGES_REQUESTED** [review]
This is just wrong hahah

- winston logger doesnt implement verbose and silly methods from I logger interface
- instead of custom filter, you should do:
```
createLogger({
levels: {
   levels: {
      error: 0, 
      warn: 1, 
      info: 2, 
      verbose: 3, 
      debug: 4, 
      silly: 5 
   },
  colors: {
     error: 'red',
     warn: 'yellow',
     info: 'green',
     verbose: 'cyan',
     debug: 'blue',
     silly: 'magenta'
  }
}
})
```

Although that are default values so I'm pretty sure you got weird behavior because of wrong level option supplied or missing methods

**Comment by GregTheGreek** [comment]
@mpetrunic verbose and silly are built into the winston logger...

**Comment by mpetrunic** [comment]
> @mpetrunic verbose and silly are built into the winston logger...

interface methods still not implemented in WinstonLogger class

I though you wanted to change colors on them

**Comment by GregTheGreek** [comment]
> interface methods still not implemented in WinstonLogger class

Ah yes you're right I'll add those. 

> I though you wanted to change colors on them

For now, I don't see a point - adding the extra log levels should suffice for now

**Comment by GregTheGreek** [comment]
updated

**Review by mpetrunic — APPROVED** [review]

File: packages/lodestar/src/logger/interface.ts [hunk 2]
```diff
@@ -6,16 +6,29 @@ export enum LogLevel {
...
 ];
 
+export const customColors = {
+  error: "red",
+  warn: "yellow",
+  info: "white",
+  verbose: "green",
+  debug: "pink",
+  silly: "purple",
+};
```
Comment (wemeetagain) [inline packages/lodestar/src/logger/interface.ts:30]: love it

**Review by wemeetagain — APPROVED** [review]

**Pull request merged**